packages/manager/modules/server-sidebar/src/order.constants.js

```diff
@@ -262,36 +262,21 @@ export const ORDER_URLS = {
         'https://www.ovhcloud.com/en/network-security/ovhcloud-connect/provider/',
     },
     dbaas_logs: {
-      FR:
-        "https://www.ovh.com/fr/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      CZ:
-        "https://www.ovh.cz/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      DE:
-        "https://www.ovh.de/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      ES:
-        "https://www.ovh.es/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      FI:
-        "https://www.ovh-hosting.fi/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      GB:
-        "https://www.ovh.co.uk/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      IE:
-        "https://www.ovh.ie/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      IT:
-        "https://www.ovh.it/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      LT:
-        "https://www.ovh.lt/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      MA:
-        "https://www.ovh.ma/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      NL:
-        "https://www.ovh.nl/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      PL:
-        "https://www.ovh.pl/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      PT:
-        "https://www.ovh.pt/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      SN:
-        "https://www.ovh.sn/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      TN:
-        "https://www.ovh.com/tn/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
+      FR: 'https://www.ovh.com/manager/dedicated/#/dbaas/logs/welcome',
+      CZ: 'https://www.ovh.com/manager/dedicated/#/dbaas/logs/welcome',
+      DE: 'https://www.ovh.com/manager/dedicated/#/dbaas/logs/welcome',
+      ES: 'https://www.ovh.com/manager/dedicated/#/dbaas/logs/welcome',
```

Comment:
Why duplicate ? If the format we need to pass is not handled maybe we should improve the way we parse constants and not duplicate them